substrate-archive/src/actors.rs

```diff
@@ -199,6 +199,7 @@ where
 		self.blocks.send(ReIndex).await?;
 		let actors = self.clone();
 		task::spawn(async move {
+			let _ = &actors;
```

Comment:
remove?!

```suggestion
```

Reply:
this is from cargo fix --edition. I would say refactoring or leaving is best, as I think this pattern saves us from a breaking change planned in behavior in this edition

Reply:
it's not a breaking change rather that the reference is unused thus it's a bug in `cargo fix` FWIW :)

//cc @insipx  ^^

Reply:
Yeah i think removing this is the play. Compiles just fine w/o it